orchagent/intfsorch.cpp

```diff
@@ -282,6 +282,13 @@ void IntfsOrch::doTask(Consumer &consumer)
                 continue;
             }
 
+            if (port.m_bridge_port_id != SAI_NULL_OBJECT_ID)
+            {
+                SWSS_LOG_ERROR("Router Intfs config on vlan member %s is not allowed", alias.c_str());
+                it = consumer.m_toSync.erase(it);
```

Comment:
I don't think we should block and discard the configuration. Instead, why don't we retry here?

Reply:
The concern is that performing retry here will introduce too many error logs and extra system overhead if the error is persistent for a long time. While lowering the log level will leave the error unnoticed most likely.